mp/src/creategameprojects.ps1

```diff
@@ -8,7 +8,7 @@ if (-NOT ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdent
 $ErrorActionPreference= 'silentlycontinue'
 # This is the path to your steamworks dev folder (depo). This should be the folder with hl2.exe inside of it.
 # Your local momentum folder will be sym linked INSIDE this folder as momentum
-$path = "E:\Steamworks\dev\Momentum"
+$path = "E:\Coding Projects\MomentumDev"
```

Comment:
You do not want to commit this, and you can actually revert this file back after running `creategameprojects.bat` as admin, since the symlink is only needed to be created once. Any future runs of `creategameprojects.bat` (when you change branch or edit any `.vpc` file) can be run as normal (non-admin).